src/components/content/injection.tsx

```diff
@@ -11,12 +18,21 @@ export interface InjectionProps {
 export function Injection({ href }: InjectionProps) {
   //TODO: define and export data types somewhere
   const [value, setValue] = React.useState<any>(undefined)
-  const [license, setLicense] = React.useState<undefined | LicenseNoticeData>(
+  const [license, setLicense] = React.useState<LicenseNoticeData | undefined>(
     undefined
   )
+  const [prettyLinks, setPrettyLinks] = React.useState<PrettyLinksContextProps>(
```

Comment:
nitpick: would probably rename that type to `PrettyLinksContextValue` or something similar since it doesn't represent the type of props of a component named `PrettyLinksContext`.